cocotb/handle.py

```diff
@@ -479,6 +479,11 @@ def __le__(self, value):
         Example:
         >>> module.signal <= 2
         """
+        warnings.warn(
+            "Setting values on handles using the ``handle <= value`` syntax will be deprecated. "
```

Comment:
I know we heard otherwise as part of this review, and I'll leave it up to you to weigh the arguments, but my 2c: Either it's deprecated, or not. A "will be deprecated" is silly, IMO.

Reply:
I changed this based on @eric-wieser's suggestion this should be a `PendingDeprecationWarning` instead of a `DeprecationWarning`, which to me say that it will become a `DeprecationWarning` in the future. I agree it should be full deprecation and don't really understand what `PendingDeprecationWarning` means. [Some core devs agree](https://discuss.python.org/t/pendingdeprecationwarning-is-really-useful/1038).

Reply:
Changed wording to `is deprecated` and changed warning to `DeprecationWarning`. I don't feel like messing around with this stuff.